gammapy/datasets/map.py

```diff
@@ -1467,12 +1468,17 @@ def stat_sum(self):
         counts, npred = self.counts.data.astype(float), self.npred().data
 
         if self.mask is not None:
-            return (
-                cash_sum_cython(counts[self.mask.data], npred[self.mask.data])
-                + prior_stat_sum
-            )
+            mask = self.mask.data == True  # noqa
```

Comment:
Not sure I fully understand what this line here does. Does it converts values to boolean ?